File: loianeTraining/javaBasico/Exer Aula 20/Ex2.java
import java.util.Random;

public class Ex2 {
    public static void main(String[] args) {
        Random random = new Random();
        int[][] matriz = new int[10][10];

        for (int line = 0; line < matriz.length; line++) {
            for (int column = 0; column < matriz.length; column++) {
                matriz[column][line] = random.nextInt(0, 10);
                System.out.print(matriz[column][line] + "  ");
            }
            System.out.println();
        }

        var maxValueColumn = Integer.MIN_VALUE;
        var minValueColumn = Integer.MAX_VALUE;
        var maxValueLine = Integer.MIN_VALUE;
        var minValueLine = Integer.MAX_VALUE;

        for (int line = 0; line < matriz.length; line++) {
            for (int column = 0; column < matriz.length; column++) {
                if (matriz[line][5] > maxValueLine) {
                    maxValueLine = matriz[column][5];
                }
                if (matriz[line][5] < minValueLine) {
                    minValueLine = matriz[column][5];
                }

                if (matriz[7][column] > maxValueColumn) {
                    maxValueColumn = matriz[7][line];
                }

                if (matriz[7][column] < minValueColumn) {
                    minValueColumn = matriz[7][line];
                }
            }
        }
        System.out.println("Coluna 7 - Maior valor: " + maxValueColumn);
        System.out.println("Coluna 7 - Menor valor: " + minValueColumn);
        System.out.println("Linha 5 - Maior valor: " + maxValueLine);
        System.out.println("Linha 5 - Menor valor: " + minValueLine);
    }
}

File: loiane-training-csharp/procedural-poo/f/Ex2.cs
using System;

class Ex2
{
    static void Main(string[] args)
    {
        int[,] matriz = new int[10, 10];

        Random random = new Random();
        int maiorValorLinha5 = int.MinValue;
        int menorValorLinha5 = int.MaxValue;
        int maiorValorLinha7 = int.MinValue;
        int menorValorLinha7 = int.MaxValue;
        for (int i = 0; i < matriz.GetLength(0); i++)
        {
            for (int j = 0; j < matriz.GetLength(1); j++)
            {
                matriz[i, j] = random.Next(0, 9);

                if (i == 5 && maiorValorLinha5 < matriz[i, j])
                {
                    maiorValorLinha5 = matriz[i, j];
                }

                if (i == 5 && menorValorLinha5 > matriz[i, j])
                {
                    menorValorLinha5 = matriz[i, j];
                }

                if (i == 7 && maiorValorLinha7 < matriz[i, j])
                {
                    maiorValorLinha7 = matriz[i, j];
                }

                if (i == 7 && menorValorLinha7 > matriz[i, j])
                {
                    menorValorLinha7 = matriz[i, j];
                }
            }
        }

        for (int i = 0; i < matriz.GetLength(0); i++)
        {
            for (int j = 0; j < matriz.GetLength(1); j++)
            {
                Console.Write(matriz[i, j] + " ");
            }
            Console.WriteLine();
        }

        Console.WriteLine("Linha 5 - Menor valor: {0} | Maior valor: {1}", menorValorLinha5, maiorValorLinha5);
        Console.WriteLine("Linha 7 - Menor valor: {0} | Maior valor: {1}", menorValorLinha7, maiorValorLinha7);
    }
}